app/routes/task_routes.py

```diff
@@ -0,0 +1,124 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+from datetime import datetime
+import requests
+from dotenv import load_dotenv
+import os
+
+
+load_dotenv()
+
+task_bp = Blueprint("tasks", __name__,url_prefix="/tasks")
+
+path = "https://slack.com/api/chat.postMessage"
+KEY = os.environ.get("API_SECRET_KEY")
```

Comment:
It would be more descriptive to say the source of the key, like SLACK_API_KEY. Soon you'll be working on a route file that has several API keys in it so we should be more specific.